api/orchestration_movers.go

```diff
@@ -452,3 +452,32 @@ func (o *datasyncOrchestrator) taskDetailsFromName(ctx context.Context, group, n
 
 	return nil, nil, apierror.New(apierror.ErrNotFound, "datasync mover not found", nil)
 }
+
+// taskRunsFromTaskArn finds a datasync task runs  based on its group/name and returns information about it
+func (o *datasyncOrchestrator) taskRunsFromTaskArn(ctx context.Context, taskArn string) ([]string, error) {
+	if taskArn == "" {
+		return nil, apierror.New(apierror.ErrBadRequest, "invalid input", nil)
+	}
+
+	// get a list of all datasync resources in the group
+	out, err := o.datasyncClient.ListDatasyncTaskExecutions(ctx, taskArn)
```

Comment:
So here we should return just the exec id, not the entire ARN, it's the last part:
`arn:aws:datasync:us-east-1:1234567890:task/task-0c1e0e879dba97ae3/execution/exec-0bdf789eb0dca66cb`

Reply:
Done, please validate.